src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -397,6 +379,26 @@ export default {
       }
     },
 
+    loadCorrection (filing, section) {
+      console.log(filing)
+      if (section) {
+        const item = {
+          type: filing.header.name,
+          filingAuthor: filing.header.certifiedBy,
+          filingDate: filing.correction.correctedFilingDate,
+          filingId: filing.header.filingId,
+          corrFilingId: filing.correction.correctedFilingId,
+          correctedFilingType: filing.correction.correctedFilingType,
+          title: `Correction - ${this.typeToTitle(filing.correction.correctedFilingType)}`,
+          comments: this.flattenAndSortComments(filing.header.comments)
+        }
+        this.filedItems.push(item)
+      } else {
+        // eslint-disable-next-line no-console
+        console.log(`ERROR - missing section in filing =`, filing)
```

Comment:
Please use apostrophes (') for normal strings and back-ticks (`) only for template literals/strings (in which you embed expressions).

So you _could_ rewrite this as:
<pre>
console.log(`ERROR - missing section in filing = ${filing}`)
</pre>

Or you could change the back-ticks to apostrophes.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
A copy paste mistake, fixed!